testing/evaluate_tests.py

```diff
@@ -113,6 +113,24 @@ def test_schema(full_response, test_value, test_name=''):
     then make sure the value is correct
     """
 
+    # Exclude word_confusions from the search below. There was an error when the word was the same as a return_json key
+    # segment['word_confusions'] =
+    #   [{'AMD': 1},
+    #   {'had': 1},
+    #    ...
+    #   {'revenue': 1},
+    # and
+    # interpreted_fields:
+    #       company_name: '@nlprocessor_company_name'
+    #       key_metrics:
+    #         revenue:
+    #
+    # the revenue key metric was never found since the word confusion {'revenue':1} was found first
+
+    # assume there is only one segment since that's how the tests are written
+    if full_response.get('segments') and full_response.get('segments')[0].get('word_confusions'):
+        del full_response['segments'][0]['word_confusions']
```

Comment:
Do we always get segments back from discovery? [We can set](https://github.com/greenkeytech/scribe/blob/master/projects/discovery/src/discovery_utils/prune_lattice.py) `PRUNE_LATTICE=True` and remove this code.

If we require word confusions we can simplify this.
```suggestion
    if "word_confusions" in full_response.get('segments', [""])[0]:
        del full_response["segments"][0]["word_confusions"]
```

Reply:
evaluate_tests.py reads the json test files that are user created

i'd want to keep logic that works with both segments and pruned-lattices

Reply:
`PRUNE_LATTICE` [removes the output](https://docs.greenkeytech.com/focus-api/nlp/#configuration_4) word confusions and shouldn't affect the input. Then we can use this variable to remove this code, right? It's clear no tests depend on the output word confusions.

Reply:
i think that would work

discovery would output `full_response` with no word_confusions

Reply:
ahh.. something might not be working as expected for the external_json that we have

the revenue test doesn't pass with PRUNE_LATTICE=True
```
integration test 2          schema              {"revenue": "$3 billion"}                                                                           
                                                {"revenue": 1}
```